nextflow_schema.json

```diff
@@ -664,6 +664,13 @@
                     "default": 0,
                     "description": "Maximum number of internal stop codons allowed in a sequence.",
                     "minimum": 0
+                },
+                "lx_allele_database": {
+                    "type": "string",
+                    "description": "Folder of locidex databases. The folder should contain a 'manifest.json' file created by locidex",
+                    "pattern": "^\\S+$",
+                    "exists": true,
+                    "format": "directory-path"
```

Comment:
null is desired, as the database can be over-ridden with the `allele_database` param negating any automated database selection.

Reply:
@apetkau Just verifying this is still the desired behaviour?

Reply:
The default should be `null`, since there is no real default that makes sense that we would be distributing with the software (in the test profile we'd use a test database, but that likely isn't intended for real usage).